### PR TITLE
Rpc patch 1

### PR DESCRIPTION
**Pull request by sginoza**
Minimum set of updates made for correctness and to conform with house style.  Examples: removed change log; moved acks to the end; changed links to include section numbers; s/section/Section; some grammar and punctuation updates

### CODE REVIEW

**Comment by sginoza** [comment]
Please disregard this PR!  It was added by mistake --  apologies!

File: README.md [hunk 1]
```diff
@@ -1,24 +1,73 @@
-# Delegated Credentials for TLS
 
-This is the working area for the IETF [TLS Working Group](https://datatracker.ietf.org/wg/tls/documents/) Internet-Draft, "Delegated Credentials for TLS".
 
-* [Editor's Copy](https://tlswg.github.io/tls-subcerts/#go.draft-ietf-tls-subcerts.html)
-* [Working Group Draft](https://tools.ietf.org/html/draft-ietf-tls-subcerts)
-* [Compare Editor's Copy to Working Group Draft](https://tlswg.github.io/tls-subcerts/#go.draft-ietf-tls-subcerts.diff)
 
-## Building the Draft
+# AUTH48 for RFC-to-be 9345 <draft-ietf-tls-subcerts-15>
```
Comment (bifurcation) [inline README.md:4]: Rather than putting your content into `README.md`, this content should go in the comments of your pull request, rather than checking it into the repo itself.  (The comment field also accepts Markdown.)  You could make the title of the PR "AUTH48 for RFC-to-be 9345", and put the content here in the comments.

If you want to have a file in the repo for some reason (e.g., so that it lives longer than one PR), please make it something separate, say `AUTH48.md`.  Then we can delete it when AUTH48 is over.

File: README.md [hunk 2]
```diff
@@ -1,24 +1,73 @@
...
+This repository is for managing document updates for RFC-to-be 9345 during AUTH48.
```
Comment (bifurcation) [inline README.md:5]: No, this repo is for the document itself :)  The *PR* is for AUTH48.

File: README.md [hunk 3]
```diff
@@ -1,24 +1,73 @@
...
+* question - a follow-up question has been added to the issue discussion. Use @mention to identify the participant who should answer. 
+* AD approval required - the resolution of the issue requires the AD to review and approve the change.
+* rfced-ready - the RFC Editor can update the document and/or close the issue based on the issue discussion.
+
+Feel free to add new issues and PRs for new edits and questions. 
+
+The editor will close the issue when it has been addressed either by accepting a pull request or simply closing the issue if no changes are needed. 
+
+## Pull Requests
+If you would like to submit changes, please create a pull request and identify one or more issues that the pull request closes in the comments.
```
Comment (bifurcation) [inline README.md:34]: This process seems backwards.  What I would expect is:

* RPC sends a PR with edits to the MD
* Editors make comments on / changes to the PR
* When the editors approve the changes, they signify this by merging the PR

File: README.md [hunk 4]
```diff
@@ -1,24 +1,73 @@
...
+* [https://www.rfc-editor.org/authors/rfc9345.txt](https://www.rfc-editor.org/authors/rfc9345.txt)
+
+Once the content of the .md file is stable and an XML file has been created, the following files will also be provided:
+* [https://www.rfc-editor.org/authors/rfc9345.html](https://www.rfc-editor.org/authors/rfc9345.html)
+* [https://www.rfc-editor.org/authors/rfc9345.pdf](https://www.rfc-editor.org/authors/rfc9345.pdf)
+* [https://www.rfc-editor.org/authors/rfc9345.xml](https://www.rfc-editor.org/authors/rfc9345.xml)
+   
+To create your own output files, use the [tools webservice](https://author-tools.ietf.org/) and select one of the output options.
+
+## Viewing Diffs
```
Comment (bifurcation) [inline README.md:50]: If we're using this repo as opposed to an RPC one, this section seems unnecessary.  We'll just use PR diffs, as in this one.

File: README.md [hunk 5]
```diff
@@ -1,24 +1,73 @@
...
+The following diff files are available via www.rfc-editor.org:
+* [https://www.rfc-editor.org/authors/rfc9345-diff.html](https://www.rfc-editor.org/authors/rfc9345-diff.html) (all changes since the document entered the RFC Editor Queue)
+* [https://www.rfc-editor.org/authors/rfc9345-rfcdiff.html](https://www.rfc-editor.org/authors/rfc9345-rfcdiff.html) (side-by-side view of all changes since the document entered the RFC Editor Queue)
+
+## Approving the Document 
+This document is being edited in Markdown.  Once the content is stable, the Markdown will be converted to XML and formatted as an RFC.  Your final approval means you approve both the content and format.
+
+To approve your RFC for publication, **please reply to the AUTH48 email** stating that you approve this RFC for publication.  Please use **REPLY ALL**, as all the parties CC’ed on the message need to see your approval.
+
+The details of the AUTH48 status of the document are here: [https://www.rfc-editor.org/auth48/rfc9345](https://www.rfc-editor.org/auth48/rfc9345)
```
Comment (bifurcation) [inline README.md:65]: I would suggest a different final approval process:

* Once all of the RPCs edit PRs have been merged, the RPC submits a "final approval" PR
* This PR only does one thing: Change the name of the doc from `draft-xxx.md` to `rfcXXXX.md`
* The editors' merging this PR signifies their approval

This way, you get a specific commit that identifies the exact Markdown that is agreed.  You could also confirm in email.

File: README.md [hunk 6]
```diff
@@ -1,24 +1,73 @@
...
+
+## About This Repo
+Note that this repo is available only during AUTH48. Before the RFC publication announcement is sent, this repo will be archived.
```
Comment (bifurcation) [inline README.md:68]: Since we're doing this in the WG repo, this paragraph is inaccurate.  We will keep this around in case of a -bis.

File: draft-ietf-tls-subcerts.md [hunk 7]
```diff
@@ -1,13 +1,15 @@
...
+submissionType: IETF
+date: January 2023
 category: std
 
 ipr: trust200902
-area: Security
-workgroup:
-keyword: Internet-Draft
+# area: Security
+# workgroup: tls
```
Comment (bifurcation) [inline draft-ietf-tls-subcerts.md:11]: Why the `#` characters here?

File: draft-ietf-tls-subcerts.md [hunk 8]
```diff
@@ -1,13 +1,15 @@
 ---
 title: Delegated Credentials for (D)TLS
 abbrev:
-docname: draft-ietf-tls-subcerts-latest
+docname: draft-ietf-tls-subcerts-15
```
Comment (bifurcation) [inline draft-ietf-tls-subcerts.md:4]: I would leave this as it is until we change the name to `rfc9345`.  Unless you need it to have -15 for some tooling to work?

File: note-well.md [hunk 9]
```diff
@@ -0,0 +1,25 @@
+# IETF Note Well
```
Comment (bifurcation) [inline note-well.md:1]: We should avoid replicating the note well in every WG repository.  The relevant links are already in `CONTRIBUTING.md`.  If you want to update that to be more like the official Note Well (is there one?), please make a separate PR that updates that file instead of adding a new one.  But this seems pretty far afield from the RPC's role?

**Review by bifurcation — COMMENTED** [review]
I have not reviewed the content changes here, aside from a few metadata questions.  Overall, I think we need some process tweaks, but I **really** like that the RPC edits are showing up in the MD file!
